cmd/commands/server/command_server.go

```diff
@@ -75,15 +75,20 @@ func (cmd *Command) Start() (err error) {
 	cmd.dialogWaiter = cmd.dialogWaiterFactory(providerID)
 	providerContact, err := cmd.dialogWaiter.Start()
 
+	publicIP, err := cmd.ipResolver.GetPublicIP()
+	if err != nil {
+		return err
+	}
+
 	// if for some reason we will need truly external IP, use GetPublicIP()
-	vpnServerIP, err := cmd.ipResolver.GetOutboundIP()
+	outboundIP, err := cmd.ipResolver.GetOutboundIP()
```

Comment:
Aren't we doing option `openvpn.ip` with this?
For those who wants has different public IP and wats to map it in router

Reply:
There is not such option at the moment. And this is a quick detection with posibility for user to take some action.